.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,15 +8,17 @@
 
 ## Changes proposed
 
-<!-- Here comes all the changes proposed through this PR -->
+<!-- List all the proposed changes in your PR -->
 
-<!-- Check all the boxes which are applicable to check the box correct follow the following conventions -->
+<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
 <!--
-[x] - Correct
-[X] - Correct
+[x] - Correct; marked as done
+[X] - Correct; marked as done
+
+[ ] - Correct; marked as **not** done
 -->
 
-## Check List (Check all the boxes which are applicable) <!-- Follow the above conventions to check the box -->
+## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->
 
 - [ ] My code follows the code style of this project.
 - [ ] My change requires a change to the documentation.
```

Comment:
maybe change/changes? (could be multiple too)

Reply:
```suggestion
- [ ] My change requires change(s) to the documentation.
```